packages/notify-client/test/clients.spec.ts

```diff
@@ -583,7 +588,8 @@ describe("Notify", () => {
         expect(updateEvent.topic).toBe(subscriptions[0].topic);
       });
 
-      it("automatically fires watchSubscriptions on init", async () => {
+      // This test needs a refactor involving mocking event emitter
```

Comment:
Could you explain what the issue here is exactly pls?

```suggestion
      // TODO: This test needs a refactor involving mocking event emitter
```

Reply:
Lmao damn, ok thanks for the context 👍